newsletter-march-2023.rst

```diff
@@ -0,0 +1,127 @@
+.. post:: March 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - March 2023
+=====================================
+
+News and updates
+----------------
+
+Here are the latest updates from our team since the :doc:`previous newsletter </newsletter-february-2023>`:
+
+- ⭐️ We passed our 10,000th issue/pull request on GitHub. And it's pretty much an equal split between the `5039 issues <https://github.com/readthedocs/readthedocs.org/issues>`__ and `4872 pull requests <https://github.com/readthedocs/readthedocs.org/pulls>`__ now registered.
+  Thanks to the whole community for building this together through code, issues, suggestions... and documentation!
+- 🌪️ Follow up: Build times have gone rapidly down after last month's introduction of parallel uploading of artifacts with `rclone`_.
+  Depending on the number of files in a build outputs,
+  build times may have gone down several seconds or several minutes.
+  For instance, a large project like `Write the Docs <https://readthedocs.org/projects/writethedocs-www/builds/>`__ has gone down from ~7 minutes to under 3 minutes.
+  If you want to see the change for your project,
+  have a look at your build times before and after **February 8**.
+- 1️⃣️ We are going to have 1 main Read the Docs website in the future.
+  The first move has been done now,
+  as readthedocs.com now points to `about.readthedocs.com <https://about.readthedocs.com>`__ instead of being a separate site for Read the Docs for Business.
+  The separate landing page on readthedocs.org will also be moving.
+- 📚️ Documentation refactor underway:
+  A regular reader of our :doc:`user docs <readthedocs:index>` might notice that our navigation has been greatly changed and now contains only the `4 Diátaxis categories <https://diataxis.fr/>`__.
+- 🧹️ We are now able to delist projects from search engines more efficiently.
+  If you find an unofficial and unmaintained project,
+  :doc:`please see our policy on unofficial and unmaintained projects <readthedocs:unofficial-projects>`.
+- 🐞 Custom 404s with Sphinx DirHTML builder `are now supported <https://github.com/readthedocs/sphinx-notfound-page/issues/215>`__.
+- 🐞 :doc:`Pull Request <readthedocs:guides/pull-requests>` builds now link directly to the preview version of the documentation changes.
```

Comment:
Hmmmmmmm :)

The link isn't working on the latest build of *this* PR. Ping @ericholscher

Reply:
Yea, it's not getting deployed til Tuesday. I don't think this is worth mentioning, since it's a bug fix, not a new feature.

Reply:
Updated the text... can decide if this is too insignificant or good marketing to make people aware that the behavior is unintentional and being fixed? It has an emoji that makes it easy to jump past for readers that aren't interested in our bug fixes.